modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: CONCEPT
+
+[id="relocation-of-openshift-jenkins-images_{context}"]
+= Relocation of OpenShift Jenkins images
+
+.What stays the same with the OpenShift Jenkins images?
+
+* The Cluster Samples Operator manages the `ImageStream` object and pod templates for operating the OpenShift Jenkins images.
+* By default, the Jenkins `DeploymentConfig` object from the Jenkins pod template triggers a redeployment when the Jenkins image changes. By default, this image is referenced by the `jenkins:2` image stream tag of Jenkins image stream in the `openshift` namespace in the `ImageStream` YAML file in the Samples Operator payload.
+* If you upgrade from {product-title} 4.10 and earlier to 4.11, the deprecated `maven` and `nodejs` pod templates are still in the default image configuration.
+* If you upgrade from {product-title} 4.10 and earlier to 4.11, the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams still exist in your cluster. To maintain these image streams, see the “So what happens with the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams in the `openshift` namespace?” section below.
+
+.What changes in the support matrix of the OpenShift Jenkins image?
+
+Each new image in the `ocp-tools-4` repository in the `registry.redhat.io` registry supports multiple versions of {product-title}. When Red Hat updates one of these new images, it is available for all versions at the same time. This is ideal, for example, when Red Hat updates an image in response to a security advisory.
+
+Previously, each Jenkins image supported only one version of {product-title} and Red Hat might not update all images simultaneously.
+
+.What additions are there with the OpenShift Jenkins and Jenkins Agent Base ImageStream and ImageStreamTag objects?
+
+By moving from an “in-payload” image stream to an image stream that references non-payload images, {product-title} can define as many image stream tags as needed. Red Hat has created a series of new image stream tags to go along with the existing `"value": "jenkins:2"` and `"value": "image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-base:latest"` image stream tags present in {product-title} 4.10 and earlier. These new image stream tags address some requests to improve how the Jenkins-related image streams are maintained.
+
+About the new image stream tags:
```

Comment:
yeah these descriptions of the new tags are much more accurate than the ones you had in https://github.com/openshift/openshift-docs/pull/47660/files#diff-c38c7d95832f84da2f9bdaec53fb837e4fb3c6499d8d436dc68348e867c18fab

Reply:
That's intentional. `modules/customizing-the-jenkins-image-stream-tag.adoc` is a procedure and should provide only enough information for the user to perform the action. In contrast, this topic, `modules/relocation-of-openshift-jenkins-images.adoc` is a `CONCEPT` topic whose purpose is to provide additional explanation about the procedure.